ComSemApp/utils.py

```diff
@@ -70,12 +74,64 @@ def transcribe(request):
             try:
                 text = r.recognize_google(audio)
                 #closes the temp files
-                os.close(in_file_handle)
-                os.close(out_file_handle)
+                close(in_file_handle)
+                close(out_file_handle)
```

Comment:
why the change?

Reply:
close was the only thing being used from os, and I figured this looked a bit cleaner